src/status_im/translations/en.cljs

```diff
@@ -186,7 +190,9 @@
    :wallet-set-up-confirm-title          "Wrote it down?"
    :wallet-set-up-confirm-description    "You won’t be able to see your 3-word transaction phrase again after this."
    :wallet-backup-seed-title             "Backup your Seed Phrase"
```

Comment:
do we still need ` :wallet-backup-seed-title  ` ?

Reply:
Probably not. I can clean them up